Conventions.sln

```diff
@@ -1,10 +1,44 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.0.0
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.4.33103.184
 MinimumVisualStudioVersion = 15.0.0.0
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GraphQL.Conventions", "src\GraphQL.Conventions\GraphQL.Conventions.csproj", "{AEAC9A4B-7D5B-48C6-86A9-E665F6DE4B6C}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Conventions", "src\GraphQL.Conventions\GraphQL.Conventions.csproj", "{AEAC9A4B-7D5B-48C6-86A9-E665F6DE4B6C}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GraphQL.Conventions.Tests", "test\GraphQL.Conventions.Tests\GraphQL.Conventions.Tests.csproj", "{5CF44C73-E5C9-4E53-A69D-4013E8AE85E5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Conventions.Tests", "test\GraphQL.Conventions.Tests\GraphQL.Conventions.Tests.csproj", "{5CF44C73-E5C9-4E53-A69D-4013E8AE85E5}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SimpleWebApp", "samples\SimpleWebApp\SimpleWebApp.csproj", "{19B85ED5-1F02-4D9D-AE9E-F265FC64C5D9}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DataLoaderWithEFCore", "samples\DataLoaderWithEFCore\DataLoaderWithEFCore\DataLoaderWithEFCore.csproj", "{B4F24672-8293-4016-AE29-304C2758D4D1}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Samples", "Samples", "{117AB27B-C4F4-4072-A857-4D2B868BEB80}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SubscriptionExample", "samples\SubscriptionsGraphQLServer\SubscriptionExample\SubscriptionExample\SubscriptionExample.csproj", "{E7DA8A82-3914-41E5-B5BE-D0BB3F234CE3}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".github", ".github", "{60A6D426-6E98-45CA-B5C4-59B27C4EFD81}"
+	ProjectSection(SolutionItems) = preProject
+		.github\FUNDING.yml = .github\FUNDING.yml
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "workflows", "workflows", "{67DFCB1B-6331-41E5-8FD3-03964526EDDD}"
+	ProjectSection(SolutionItems) = preProject
+		.github\workflows\build.yml = .github\workflows\build.yml
+		.github\workflows\formatting.yml = .github\workflows\formatting.yml
+		.github\workflows\publish.yml = .github\workflows\publish.yml
+		.github\workflows\test.yml = .github\workflows\test.yml
+		.github\workflows\wipcheck.yml = .github\workflows\wipcheck.yml
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".Solution Items", ".Solution Items", "{5A8618B7-7299-4507-A665-CC9CEE8B4674}"
+	ProjectSection(SolutionItems) = preProject
+		global.json = global.json
```

Comment:
I doubt about this file. In other OSS projects that have it I should always touch/fix it to build code locally.

Reply:
I do not know what it is there for; I would delete it.  I would propose the change in a separate PR so @tlil can review.